lib/resque/plugins/exponential_backoff.rb

```diff
@@ -80,10 +80,11 @@ def retry_limit
 
       # Selects the delay from the backoff strategy
       #
+      # @param _ [Exception] unused exception argument for signature parity
       # @return [Number] seconds to delay until the next retry.
       #
       # @api private
-      def retry_delay
+      def retry_delay(_ = nil)
```

Comment:
@jzaleski let me know if this is what you wanted. I thought that the leading underscore version was more communicative, but if you're fine not caring about the args at all, I'd suggest just:

```ruby
def retry_delay(*)
```

Reply:
I think this is fine as is, `_` communicates the intent effectively and is more specific than the wildcard suggestion.